src/lib/constant_mappings.cpp

```diff
@@ -139,4 +142,16 @@ const boost::bimap<DataType, std::string> data_type_to_string =
                  return map;
                });
 
+const boost::bimap<EncodingType, std::string> encoding_type_to_string = make_bimap<EncodingType, std::string>({
+    {EncodingType::DeprecatedDictionary, "Dictionary (Deprecated)"},
```

Comment:
Shouldn't that be gone in the current master?

Reply:
Do you mean the deprecated dictionary column? Unfortunately not yet :(